llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -2955,40 +2956,35 @@ void LoopAccessInfo::collectStridedAccess(Value *MemAccess) {
   // of various possible stride specializations, considering the alternatives
   // of using gather/scatters (if available).
 
-  const SCEV *MaxBTC = PSE->getSymbolicMaxBackedgeTakenCount();
+  const SCEV *BTC = PSE->getSymbolicMaxBackedgeTakenCount();
```

Comment:
Small caveat: I think this should be `getBackedgeTakenCount`, the exact version, and not the symbolic-max version. However, it's not going to be easy to cook up a test to support this change, but I can think about it in a follow-up.